DataFormats/ParticleFlowReco/interface/PFRecHit.h

```diff
@@ -7,20 +7,16 @@
 #include <iostream>
 
 #include "DataFormats/Math/interface/Point3D.h"
-#include "Rtypes.h" 
 #include "DataFormats/Math/interface/Vector3D.h"
 #include "Math/GenVector/PositionVector3D.h"
 #include "DataFormats/ParticleFlowReco/interface/PFLayer.h"
 #include "DataFormats/ParticleFlowReco/interface/PFRecHitFwd.h"
 
-//C decide what is the default rechit index. 
-//C maybe 0 ? -> compression 
-//C then the position is index-1. 
-//C provide a helper class to access the rechit. 
-
 #include "DataFormats/CaloRecHit/interface/CaloRecHit.h"
 #include "DataFormats/Common/interface/RefToBase.h"
 
+#include "Geometry/CaloGeometry/interface/CaloCellGeometry.h"
```

Comment:
I checked the dependencies in Geometry/CaloGeometry and what I found are
```
CLHEP/Geometry
DataFormats/CaloTowers
DataFormats/DetId
DataFormats/EcalDetId
DataFormats/GeometryVector
DataFormats/HcalDetId
DataFormats/Math
FWCore/MessageLogger
FWCore/Utilities
Math
```
All of which are OK for FWLite.